### PR TITLE
#75 fix cleanup ops

### PR DESCRIPTION
**Pull request by jmp3833**
No cleanup options were being sent to the backend, and once they were due to updates in linguine-node,  a good majority of them were broken. There were a few changes made to ensure each cleanup task works.: 

Changelog 
--
* For stemming and lemmatization, tokenized contents were being combined back together without spaces, leading to sentenceslikethisbeingtreatedaswords. 

* When a cleanup option that required tokenization to work (such as stemming) was run, the corpus text was overwritten with the tokenized contents, thus overwriting all other previous cleanups that were run. An example would be tokenizing a body of text and then running a remove caps cleanup on it, the results were not showing that the cleanup step happened. In order to mitigate this tokenization is now redone whenever a new cleanup task is done in the analysis pipeline.

### CODE REVIEW

File: linguine/ops/stem.py [hunk 1]
```diff
@@ -8,30 +8,18 @@
 from nltk.stem.snowball import EnglishStemmer
```
Comment (kmp3325) [inline linguine/ops/stem.py]: We can nix the imports here too.

File: linguine/ops/stem.py [hunk 2]
```diff
@@ -8,30 +8,18 @@
...
         for corpus in data:
+            corpusString = ""
             corpus.tokenized_contents = [porter.stem(word) for word in corpus.tokenized_contents]
-            corpus.contents = ''.join(corpus.tokenized_contents)
-        return data
\ No newline at end of file
+            for index, word in enumerate(corpus.tokenized_contents):
+                corpusString += corpus.tokenized_contents[index] + " "
+            
+            print(corpusString)
```
Comment (kmp3325) [inline linguine/ops/stem.py]: Probably want to remove this print so the output isn't crazy in PROD.

**Comment by kmp3325** [comment]
That's quite the bug they had there, took me a bit to wrap my head around it.  It seems like this is a good way to combat it for now, although I hate having to re-run tokenization with every single cleanup task.  There should really just be a distinction between which cleanups require tokenization and which do not, but making that distinction would be a lot of work for little gain in efficiency it seems.  

Anyways, I made a few comments, but otherwise this LGTM.

File: linguine/ops/stem.py (hunk 2 above)
Comment (jmp3833) [inline linguine/ops/stem.py]: Good catch! Sorry I must have missed that one. Committing now

**Comment by jmp3833** [comment]
@kmp3325 I agree that it's unnecessarily resource intensive to re run the tokenization step when we don't need it. Since there's some time left in this sprint I can take care of making this analysis-specific before merging

**Pull request merged**